README.md

```diff
@@ -1,4 +1,95 @@
 # vManage-client
 [![Python3.8](https://img.shields.io/static/v1?label=Python&logo=Python&color=3776AB&message=3.8)](https://www.python.org/)
 
-vManage client is a package for creating simple and parallel automatic requests via official vManageAPI. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being ran, you just need a connection to any vManage.
\ No newline at end of file
+vManage client is a package for creating simple and parallel automatic requests via official vManageAPI. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being run, you just need a connection to any vManage.
+
+## Installation
+```console
+pip install vmngclient
+```
+
+## Hello world example
+
+<details>
+    <summary>Python (click to expand)</summary>
+
+```Python
+from vmngclient.session import create_vManageSession
+
+
+base_url = "sandbox-sdwan-2.cisco.com/"
+username = "devnetuser"
+password = "RG!_Yw919_83"
+session = create_vManageSession(url=base_url, username=username, password=password)
+
+
+>>> "Logged as devnetuser. The session type is SessionType.TENANT"
+>>> {'title': 'Cisco vManage', 'version': '20.4.2.1', 'applicationVersion': '20.4R-vbamboo-16-Dec-2021 19:07:17 PST', 'applicationServer': 'vmanage', 'copyright': 'Copyright (c) 2022, Cisco. All rights reserved.', 'time': '2022-12-01 13:45:44', 'timeZone': 'UTC', 'logo': '/dataservice/client/logo.png'}
+```
+</details>
+
+
+## User creation example
+
+<details>
+    <summary>Python (click to expand)</summary>
+
+```Python
+from vmngclient.api.administration import UserAlreadyExistsError, UserApi
+from vmngclient.dataclasses import User
+from vmngclient.session import Session
+
+user_api = UserApi(provider_session)
```

Comment:
Hopefully passing here `provider_session` is clear enough for future users, otherwise any comment would be valuable